app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -91,7 +92,13 @@ const useStyles = makeStyles((theme) => ({
     paddingRight: 0,
   },
   nav: {
+    alignItems: "center",
```

Comment:
Safari stretches the nav bar height by default without this. Not sure why other browsers don't do this as default value for `align-items` is `stretch` so technically Safari was right... 🤷